test/GammaPoolERC721.behavior.ts

```diff
@@ -401,7 +401,7 @@ export function shouldBehaveLikeERC721(errorPrefix: any) {
           it('reverts', async function () {
```

Comment:
Let's use just `describe` instead of `context` altogether for consistency. Instead of writing the different results for the test case, use `describe` for a subcomponent and then `it` for the different test cases prefixed by "should . . ."